src/codegen.cpp

```diff
@@ -3484,25 +3538,26 @@ static size_t emit_masked_bits_compare(callback &emit_desc, jl_datatype_t *aty,
 static Value *emit_bits_compare(jl_codectx_t &ctx, jl_cgval_t arg1, jl_cgval_t arg2)
 {
     ++EmittedBitsCompares;
+    jl_value_t *argty = (arg1.constant ? jl_typeof(arg1.constant) : arg1.typ);
```

Comment:
Is this a related change?

Reply:
No, you are just seeing https://github.com/JuliaLang/julia/pull/55770 in the diff. Apparently this needs a rebase to drop that from here now